pong.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "gameobject.h"
+
+#include <vector>
+
+class Pong
+{
+private:
+    std::vector<std::unique_ptr<GameObject>>mGameObjects {};
```

Comment:
Now we're std-libbing 😎 

the main downside is these heinous lengthy typenames withe a billion zillion templates. Small price to pay but still annoying.

If you ever get an especially bad one, I recommend using `using`

```
     std::vector<std::unique_ptr<std::vector<std::shared_ptr<GameObject>>mGameObjectsCollection

...

using GameObjectSharedPtr = std::shared_ptr<GameObject>;
using GameObjectSharedPtrVector = std::vector<GameObjectSharedPtr>;
using GameObjectCollection = std::unique_ptr<GameObjectSharedPtrVector>;

GameObjectCollection mGameObjectsCollection



```

Reply:
Yeah that looks like a real mouthful. Will keep in mind!